app/linkedList.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Created by tatsiak on 4/22/17.
+ */
+'use strict';
+
+class Node {
+    constructor(value, previous = null) {
+        this._value = value;
+        this._previous = previous;
+        this._next = null;
+    }
+}
+class LinkedList {
+    constructor() {
+        this._start = null;
+        this._end = null;
```

Comment:
No comments to logic of the list. But after adding first element both _start and _end became the same, so maybe as improvement, you can keep only _start element as whole list, because if the list consists only one element - there is no reason to keep 2 equal objects.